modules/imgproc/test/test_filter.cpp

```diff
@@ -2355,5 +2355,16 @@ TEST(Imgproc, filter_empty_src_16857)
     EXPECT_TRUE(dst2.empty());
 }
 
+TEST(Imgproc_GaussianBlur, regression_11303)
+{
+    cv::Mat dst;
+    int width = 2115;
+    int height = 211;
+    double sigma = 8.64421;
+    cv::Mat src(cv::Size(width, height), CV_32F, 1);
+    cv::GaussianBlur(src, dst, cv::Size(), sigma, sigma);
+    EXPECT_EQ(0, cv::norm(src, dst, NORM_L2));
```

Comment:
This check may fail.

I observed problem by this run:
```
valgrind ./bin/opencv_test_imgproc --gtest_filter=*regression_11303 --test_threads=3
...
Intel(R) IPP version: ippIP AVX2 (l9) 2020.0.0 Gold (-) Oct 19 2019
...
[ RUN      ] Imgproc_GaussianBlur.regression_11303
/home/alalek/projects/opencv/dev/modules/imgproc/test/test_filter.cpp:2366: Failure
Expected equality of these values:
  0
  cv::norm(src, dst, NORM_L2)
    Which is: 7.96355e-05
[  FAILED  ] Imgproc_GaussianBlur.regression_11303 (5123 ms)
```

Also check fails in Debug builder (without IPP):

```
[ RUN      ] Imgproc_GaussianBlur.regression_11303
/build/precommit_linux64_no_opt/3.4/opencv/modules/imgproc/test/test_filter.cpp:2366: Failure
Expected equality of these values:
  0
  cv::norm(src, dst, NORM_L2)
    Which is: 7.96355e-05
[  FAILED  ] Imgproc_GaussianBlur.regression_11303 (252 ms)
```

This check is very strong. We could relax this, e.g:

```
EXPECT_LE(cv::norm(src, dst, NORM_L2), 1e-3);
```

Reply:
Fixed